iModelCore/Bentley/Bentley/DateTimeConverter.cpp

```diff
@@ -369,6 +369,136 @@ BentleyStatus DateTimeConverter::ComputeLocalTimezoneOffsetFromUtcTime(int64_t&
 std::regex* DateTimeStringConverter::s_dtRegex = nullptr;
 std::regex* DateTimeStringConverter::s_todRegex = nullptr;
 
+//---------------------------------------------------------------------------------------

```

Comment:
We should have tests comparing these new methods to the existing printf methods

Reply:
yes there is a lot of test. It was written by Krischan so well tested.